ui_espace_pro/src/common/utils/dateUtils.js

```diff
@@ -2,6 +2,23 @@ import dayjs from "dayjs"
 import relativeTime from "dayjs/plugin/relativeTime"
 import { NBR_EXPIRATION_J7 } from "../contants"
 
+export const formatDate = (date) => dayjs(date).format("YYYY-MM-DD")
+
+export const getCurrentDate = (separator = "/") => {
+  let newDate = new Date()
+  let date = newDate.getDate()
+  let month = newDate.getMonth() + 1
+  let year = newDate.getFullYear()
+  return `${year}${separator}${month < 10 ? `0${month}` : `${month}`}${separator}${date}`
+}
+
+export const getCurrentHourMinute = (separator = "h") => {
+  let newDate = new Date()
+  let minutes = newDate.getMinutes()
+  let hour = newDate.getHours()
```

Comment:
```suggestion
  const newDate = new Date()
  const minutes = newDate.getMinutes()
  const hour = newDate.getHours()
```

Or you can use dayjs instead